geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1518,15 +1589,18 @@ def get_alg_xarray(
             area_def,
         )
 
-        # Now apply the intepolator after applying the algorithm.
-        final_xarray = apply_interp_after_alg(
-            alg_xarray,
-            interp_plugin,
-            interp_args,
-            prod_plugin,
-            area_def,
-            processed_xarrays,
-        )
+        if prod_plugin.family in ["algorithm_interpolator_colormapper"]:
```

Comment:
Only apply interpolation after algorithm if interpolators are required for product family